### ISSUE TITLE
Transactions?

### ISSUE THREAD

**Issue by astro** [orlandov/node-sqlite#1]
I'm experiencing very low INSERT performance. http://www.sqlite.org/speed.html advises to use transactions.

`db.query("BEGIN; INSERT ...; INSERT ...; INSERT ...; COMMIT;", [], function() { });` doesn't work. Passing the SQL queries as individual serial queries doesn't work either. Any alternative to `query()`?

**Comment by orlandov** [comment]
Yeah there's a lot that needs to be optimized there... I'd recommend using the low level node bindings. Prepare your own statements, bind your placeholder variables and then step through them as necessary. Have a look in sqlite.js for how to do this, it's not too difficult.

**Comment by Pita** [comment]
I wanna do the same as @astro

**Comment by Pita** [comment]
I tried to insert multiple rows with one insert. That creates issue #32 :/
